src/components/Compose.js

```diff
@@ -46,12 +47,16 @@ function Compose(props) {
   const input = useRef();
 
   useEffect(() => {
-    const onKeyDown = (e) => {
+    const onKeyDown = e => {
       if (!e.key || e.metaKey || e.ctrlKey) {
         return;
       }
 
       if (input.current && !isPickerVisible) {
+        // don't overwrite if something already typed
+        if (e.keyCode === 38 && !message) {
```

Comment:
This has been a big surprise for me too, but [keyCode](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/keyCode) is deprecated. Could you use `.code` here? (Not sure if it implies changing the `38` to something else.